src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlCodeGeneration.cs

```diff
@@ -687,16 +684,6 @@ private SourceText GenerateGlobalResources(IEnumerable<XamlFileDefinition> files
 			writer.AppendLineIndented("// <autogenerated />");
 			AnalyzerSuppressionsGenerator.GenerateCSharpPragmaSupressions(writer, _analyzerSuppressions);
 
-			// If a failure happens here, this means that the _isWasm was not properly set as the DefineConstants msbuild property
-			// was not populated. This can happen when the property is set through a target with the "CreateProperty" task, and the
-			// Uno.SourceGeneration tasks do not execute this task properly.
-			if (!_isWasm)
-			{
-				writer.AppendLineIndented("#if __WASM__");
-				writer.AppendLineIndented("#error invalid internal source generator state. The __WASM__ DefineConstant was not propagated properly.");
-				writer.AppendLineIndented("#endif");
-			}
```

Comment:
This is not going to happen, ever. It's just an unnecessary generated code. From the comment above, this used to be an issue specific to Uno.SourceGeneration which is no longer used.

Now, the compiler will always pass us a correct value for DefineConstants.